docs/en/writing-running-appium/tutorial/swipe/ios-picker-wheels-mobile.md

```diff
@@ -0,0 +1,78 @@
+## iOS pickerWheels: Slow 'mobile:selectPickerWheelValue' example
+
+When '.setValue()' fails we can use 'mobile:selectPickerWheelValue' way.
+
+This mobile method allows to select next or previous value.
+
+You should change pickerWheel to next/previous and check it new value in
```

Comment:
It is necessary to keep changing picker wheel values until the current one is equal the desired one or the count of retries exceeds.